snet_cli/arguments.py

```diff
@@ -216,15 +213,19 @@ def add_client_options(parser):
 
 def add_registry_options(parser):
     parser.set_defaults(cmd=RegistryCommand)
-    with open(Path(__file__).absolute().parent.joinpath("resources", "contracts", "Registry.json")) as f:
-        contract_json = json.load(f)
-        parser.set_defaults(contract_json=contract_json)
+    contract_json = get_contract_json(
+        Path(__file__).absolute().parent.joinpath("resources", "contracts", "IRegistry.json"),
+        Path(__file__).absolute().parent.joinpath("resources", "contracts", "RegistryImpl.json")
+    )
+    parser.set_defaults(contract_json=contract_json)
 
     add_contract_identity_arguments(parser, [("", "registry_at")])
 
     subparsers = parser.add_subparsers(title="registry commands", metavar="COMMAND")
     subparsers.required = True
 
+
+    # Warning: none of these commands with the new Registry
```

Comment:
none of these commands *work* with the new registry?

Reply:
Fixed